cpu/sam0_common/Kconfig

```
@@ -21,6 +21,7 @@ config CPU_COMMON_SAM0
     select HAS_PERIPH_UART_MODECFG
     select HAS_PERIPH_UART_NONBLOCKING
     select HAS_PERIPH_UART_RECONFIGURE
+    select HAS_PERIPH_UART_RX_START
```

I think this is more descriptive:

```suggestion
    select HAS_PERIPH_UART_RXSTART_IRQ
```